TreeDB/docs/spec/verification.md

```
@@ -1688,3 +1688,11 @@ Coverage:
 The native-wire server does not exist yet. R0 follow-up work must add
 broader negative conformance fixtures, drift tests, and direct collection parity
 tests before claiming native-wire v1 server support.
+# Vector partition M1 verification
```

markdownlint-cli2 (0.23.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Above

(MD022, blanks-around-headings)